### PR TITLE
feat: manage incomplete nic

### PR DESCRIPTION
**Pull request by Jisay**
Require:
- [x] ovh-ux/ng-ovh-sso-auth#21 to be released

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1725,6 +1725,14 @@
     shuffle-seed "^1.1.6"
     titleize "^1.0.1"
 
+"@ovh-ux/manager-server-sidebar@^0.4.2":
```
Comment (cbourgois) [inline yarn.lock]: The version in the repository is `0.5.0`. Should not be in yarn.lock.

Comment (cbourgois) [reply yarn.lock]: Please rebase on `develop` (changes done in https://github.com/ovh-ux/manager/commit/8d5d59e25b5119d8d53d19a9b5907f746699d161)

File: packages/manager/apps/sign-up/src/index.js [hunk 2]
```diff
@@ -62,6 +62,9 @@ angular
     TranslateServiceProvider.setUserLocale(userLocale);
     $translateProvider.use(userLocale);
   })
+  .config(/* @ngInject */ (ssoAuthenticationProvider) => {
+    ssoAuthenticationProvider.allowIncompleteNic(true);
```
Comment (antleblanc) [inline packages/manager/apps/sign-up/src/index.js]: Is the `Public Cloud` application should also enable incomplete nic?

Comment (Jisay) [reply packages/manager/apps/sign-up/src/index.js]: As nobody knows exactly what can happen with incomplete nic, all applications do not allow incomplete nic before somebody detect that there is no risk to allow incomplete nics :-)

File: yarn.lock (hunk 1 above)
Comment (Jisay) [reply yarn.lock]: done, thx

File: packages/manager/apps/sign-up/src/index.js (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/sign-up/src/index.js]: It should be the other way round therefore `allowIncompleteNic(false)`

Comment (Jisay) [reply packages/manager/apps/sign-up/src/index.js]: Not for sign-up. The /auth page will create incomplete nics, then the creation script will redirect the incomplete nic to the sign-up app page. The app have to allow `incomplete` nics in order to complete them.

Comment (marie-j) [reply packages/manager/apps/sign-up/src/index.js]: Oopsie I misread, my bad

**Pull request merged**